samples/samples/backup_sample.py

```diff
@@ -35,23 +35,25 @@ def create_backup(instance_id, database_id, backup_id, version_time):
         backup as backup_pb
 
     spanner_client = spanner.Client()
-    instance = spanner_client.instance(instance_id)
-    database = instance.database(database_id)
+    instance_admin_api = spanner_client.instance_admin_api
```

Comment:
Do we really need both `instance_admin_api` and `database_admin_api`?

Its a bit weird that customers need to create both these clients even though all they need is an utility method - `instance_path`.